docs/development/track-user-behaviour.md

```diff
@@ -0,0 +1,26 @@
+# Track user behaviour
+
+*Note, the key library here - `tracker-js` is not well-documented at the time of
+writing, but the hope is that this will improve soon.*
+
+## Track clicks:
+
+* add a data-link-name attribute to your component and the `ophan-tracking.js`
```

Comment:
I think you need `data-component` as well to make sure that you get the [`referringComponent`](https://github.com/guardian/ophan/blob/75b86abcce07369c8998521399327d436246c016/tracker-js/assets/coffee/ophan/click-path-capture.coffee#L45) as well.